README.md

```diff
@@ -121,6 +121,28 @@ struct Foo
 }
 ```
 
+Closing braces should always be placed on a new line. No exceptions should be done, with regards of the numer of enclosed statements.
```

Comment:
How about "Closing braces should always be placed on a new line regardless of the numer of enclosed statements"

Reply:
MUCH better, thanks!